src/seedu/addressbook/data/person/Address.java

```diff
@@ -6,15 +6,12 @@
  * Represents a Person's address in the address book.
  * Guarantees: immutable; is valid as declared in {@link #isValidAddress(String)}
  */
-public class Address {
+public class Address extends Contact {
```

Comment:
you are not really calling `super()` constructor to set the values. think how that can be done to push more functions to the parent class.